### PR TITLE
Add Google Maven support

### PR DESCRIPTION
**Pull request by johnbintz-tidelift**

### CODE REVIEW

**Review by kjmahalingam — APPROVED** [review]
Given the changes in the other PRs and my reading of the depper README, this looks very reasonable, though I want to note I have limited context here. If you're confident this is all the changes that are necessary, then that's good with me, but otherwise feel free to wait for another approval!

**Review by mikeyoung85 — APPROVED** [review]
I don't know a lot about Depper but I think this is all that is needed

**Comment by jsonperl** [comment]
Looks good to me too ;)

**Pull request merged**
